gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -18,7 +18,10 @@ struct SkipList {
 
         using pointers = std::pair<SkipListElement*, SkipListElement*>;
         using height_array = sequence<pointers>;
-        using values_array = sequence<sequence<sequence<std::pair<uintE, uintE>>>>;
+        using edge_type = std::pair<uintE, uintE>;
```

Comment:
Can you add a comment here? What do the two values in the pair represent? Is it source and destination vertex ids? 

(Glad to see the use of semantic type names and using declarations! :smile: )